c++ training/LinkedLists/LinkedLists/Stack.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+#include "SinglyLinkedList.h"
+
+template<typename T>
+class TStack
+{
+private:
+	TSLList<T> List;
+
+public:
+	const T& Top() const
+	{
+		return List.GetHead();
+	}
+	//---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+	T& Top()
+	{
+		return List.GetHead();
+	}
```

Comment:
Everything good, everything corrent, i'm so glad. Suena mejor en español xd

Reply:
perhaps everything is correct or everything seems correct haha!